Rhinoceros_Engine/Convert/ToBHoM.cs

```diff
@@ -348,16 +348,60 @@ public static BHG.BoundingBox ToBHoM(this RHG.Box box)
 
         /***************************************************/
 
+        public static BHG.ISurface ToBHoM(this RHG.BrepFace face)
```

Comment:
Can we factorise this into two methods and call them from here? `private static BHG.PlanarSurface ToBHoMPlanarSurface(this RHG.BrepFace face)` and `private static BHG.PlanarSurface ToBHoMNurbsSurfaceSurface(this RHG.BrepFace face)` .

That should follow the same pattern we use for `Brep`s.

Reply:
Following your suggestion, I've divided the method in two and cleaned up the code - I hope you will like it more now.

Reply:
Thanks @pawelbaran ! 

Does the conversion table](https://github.com/BHoM/Rhinoceros_Toolkit/wiki/BHoM---Rhinoceros-conversion-table) require modificaions?

Reply:
I am not sure - I put it as a to do after the PR gets merged (did not want to change the documentation before the code actually changes). Would you like me to action beforehand?

![image](https://user-images.githubusercontent.com/26874773/68931924-d7c6be00-0791-11ea-8597-43455711dfee.png)

Reply:
No, no worries - again - you're working across a lot of prs, do what it makes you feel more comfortable. The only thing is to keep that synchronised with the code, but it can be outdated for a couple of days if there is an issue raised - no probs.